lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/RunningWorkers.java

```diff
@@ -132,6 +138,13 @@ void launchTasklet(final Tasklet tasklet) {
           }
         }
 
+        if (cancelledTasklets.contains(tasklet.getId())) {
+          tasklet.cancelled();
+          cancelledTasklets.remove(tasklet.getId());
+          LOG.log(Level.WARNING, "Cancelled tasklet " + tasklet.getId());
```

Comment:
Should this be `WARNING`? I think the user requested to cancel the Tasklet, so wouldn't it be a desired result?